docs/kaneai-app-test-writing-guidelines.md

```diff
@@ -0,0 +1,129 @@
+---
+id: kane-ai-app-test-writing-guidelines
+title: KaneAI App Agent - Guidelines for Writing Instructions
+hide_title: false
+sidebar_label: App Agent Instruction Guide
+description: Learn how to write instructions for running the kane ai app agent smoothly and without any problem
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai App test
+  - kaneai guidelines
+  - instructions writings
+url: https://www.lambdatest.com/support/docs/kane-ai-app-test-writing-guidelines
+site_name: LambdaTest
+slug: kane-ai-app-test-writing-guidelines/
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI App Test",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-app-test-writing-guidelines"
+        }]
+      })
+    }}
+></script>
+# Best Practices for Authoring Mobile Native App Tests in KaneAI
+
+Authoring mobile native app tests in KaneAI can be significantly enhanced by following these best practices. These ensure improved stability, reliability, and accuracy of the generated steps and ultimately, the automation.
+
+## OTP and PIN Fields
+
+For OTP or PIN fields where each digit is in a separate box:
+
+- Use the keyword: `OTP`
+- Example: `Enter OTP "1234"`
+
+## Partially Clickable Text (Spannable Text)
+
+For labels or descriptions containing partially clickable links/texts:
+
+- Use the keyword: `spannable text`
+- Example: `Click on spannable text: "Terms and Conditions" in bottom info modal`
+
+## Wheels, Sliders, Seekbars, and Date Pickers
+
+Supported element classes:
+
+- **Android**: `android.widget.NumberPicker`, `android.widget.SeekBar`
+- **iOS**: `XCUIElementTypePickerWheel`, `XCUIElementTypePicker`, `XCUIElementTypeDatePicker`, `XCUIElementTypeSlider`
+
+Use instruction:
+
+- `Enter "2" in first column`
+
+If issues arise:
+
+- Use Appium Inspector or RD App Live UI Inspector to verify if the element class is supported.
```

Comment:
Use LambdaTest App Live (UI Inspector)[https://www.lambdatest.com/support/docs/ui-inspector/] to verify if the element class is supported.